tests/behaviour/typeql/steps.rs

```diff
@@ -288,13 +288,13 @@ generic_step_impl! {
                 for table_row in &step_table {
                     if match_answer_concept(
                         context,
-                        table_row.get(&Context::GROUP_COLUMN_NAME.to_string()).unwrap(),
+                        table_row.get(&Context::GROUP_COLUMN_NAME).unwrap(),
```

Comment:
`GROUP_COLUMN_NAME` is already a reference. If you run clippy, it will point out a lot of problems like this.